TwitchLib.Api.Helix/Extensions.cs

```diff
@@ -40,5 +41,28 @@ public Task<GetExtensionTransactionsResponse> GetExtensionTransactionsAsync(stri
         }
 
         #endregion
+
+        #region GetExtensionLiveChannels
+
+        public Task<GetExtensionLiveChannelsResponse> GetExtensionLiveChannelsAsync(string extensionId, int first = 20, string after = null, string accessTokenOrApplicationAccessToken = null)
```

Comment:
why not just access token? I mean I get the intention but it makes the variable name way too long imo and we just merged your PR that unified that variable name to accessToken

Reply:
The point here was to differentiate between "regular" `accessToken` and one accepting both. The naming was intentional here. There is a single endpoint that I left with that name for the same reason.